client/src/containers/funder/component.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Wrapper from 'containers/wrapper';
+
+import FundersList from './list';
+import OverviewFunder from './overview';
+
+const Funder = () => {
+  return (
+    <Wrapper>
+      <div className="pb-20 space-y-16">
+        <OverviewFunder />
```

Comment:
```suggestion
        <FunderOverview />
```
Just to maintain the consistency across the platform